demos/m4/overlays/dev/pipeline_run.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: tekton.dev/v1beta1
+kind: PipelineRun
+metadata:
+  name: build-m4-component
+  labels:
+    pipelines.appstudio.openshift.io/type: build
+    build.appstudio.openshift.io/component: m4-component
+  namespace: demo
+spec:
+  pipelineRef:
+    name: hello-world
```

Comment:
rename to be m4-release-pipeline

Reply:
I'll rename it to `m4-build-pipeline` as this is not release but build.